src/hangar/external/preferred_plugins.ini

```diff
@@ -0,0 +1,54 @@
+[jpg]
```

Comment:
Not sure why this file exists... can't we get all this info from what's provided in each plugins `.ini` file?

Reply:
These are user preferences. We need to make user preferences stateful and hence this file. We don't get this information from plugin's `ini`

Reply:
Resolved from vid chat discussion. User plugins may implement `.ini` file (which we will parse) or dict mapping in same `ini` style defined in `entry_points` hook.